### PR TITLE
Docs: READMEs for component framework integrations

### PR DESCRIPTION
**Pull request by bholmesdev**
## Changes

- Add READMEs to our component framework integrations:
  - `@astrojs/react`
  - `@astrojs/preact`
  - `@astrojs/solid`
  - `@astrojs/vue`
  - `@astrojs/svelte`
  - `@astrojs/lit`
- Includes experimental `astro add` and manual installation instructions
- Adds "getting started" section to link directly to UI framework docs
- Includes lit element specific instructions from renderer docs

## Testing

N/A

## Docs

That's what I'm doing boss!

### CODE REVIEW

**Comment by sarah11918** [comment]
How much, if at all, do we care about allll the stuff going on in the phrase:

"There's a couple ways..."  ?

:)

**Review by matthewp — APPROVED** [review]

**Comment by bholmesdev** [comment]
> How much, if at all, do we care about allll the stuff going on in the phrase:
> 
> "There's a couple ways..." ?
> 
> :)

Okay fine "there's TWO ways" 😆

File: packages/integrations/lit/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This **[Astro integration][astro-integration]** enables server-side rendering an
 
 ## Installation
 
-There's a couple ways to install integrations. Let's try the most convenient option first!
+There's two ways to add integrations to your project. Let's try the most convenient option first!
```
Comment (natemoo-re) [inline packages/integrations/lit/README.md:7]: Oh no, I'm going to be the pedant. 😬 

```suggestion
There are two ways to add integrations to your project. Let's try the most convenient option first!
```

**Review by natemoo-re — COMMENTED** [review]
Looks good other than a grammar nit!

File: packages/integrations/lit/README.md (hunk 1 above)
Comment (bholmesdev) [reply packages/integrations/lit/README.md:7]: I'm... very disappointed in myself.

**Pull request merged**